### DISCUSSION TITLE
Persistently configure TST for window

### DISCUSSION THREAD

**Discussion by 31337-4554551n** [piroor/treestyletab#3344] [Q&A] [answered]
I have firefox opening on startup.
When this happens, TST across all the firefox windows are either closed or open -same on all.
Is there a way that when firefox opens and restores sessions, to have some windows have TST open, and others to have it closed instead of applying the same setting to all windows.

I can then proceed to close/open them on individual windows as I like, and it persists for the session. But once firefox restarts, they will all be set alike

**Comment by irvinm** [comment, marked as answer]
> I have firefox opening on startup. When this happens, TST across all the firefox windows are either closed or open -same on all. Is there a way that when firefox opens and restores sessions, to have some windows have TST open, and others to have it closed instead of applying the same setting to all windows.
> 
> I can then proceed to close/open them on individual windows as I like, and it persists for the session. But once firefox restarts, they will all be set alike

This isn't TST related ... this is just how the Firefox sidebar works.  I am not sure if there is a way to do what you want to do, but it might be a good question to ask over at [r/Firefox](https://www.reddit.com/r/firefox/).
